kuma/settings/local.py

```diff
@@ -29,3 +29,6 @@
 # Don't cache non-versioned static files in DEBUG mode
 if DEBUG:
     WHITENOISE_MAX_AGE = 0
+
+# Send tasks to celery workers
+CELERY_ALWAYS_EAGER = False
```

Comment:
I've tested with both ``CELERY_ALWAYS_EQUAL=True`` and ``=False``. KumaScript works with both, and in both KumaScript seems to render the n-1 version. I'll take this out so that it can be controlled from the environment like 12factor intended.